docs/Developers/Newcomers_Guide.rst

```diff
@@ -97,104 +104,95 @@ Step 2. Grab an Invitation to the Organization
 ----------------------------------------------
 
 Let us know on gitter that you are interested in contributing and ask for an
-invitation to our org. This is your first step towards contributing.
-A maintainer will command ``cobot`` (our gitter bot) to invite
-you and be part of the Newcomer team.
+invitation to our org.
+This is your first step towards contributing.
+A maintainer will command ``cobot`` (our gitter bot) to invite you and be part
+of the Newcomer team.
 The invitation will be sent by mail and you will have to accept
-it to join. If you don't find the invitation, accept it `here <https://github.com/coala>`__.
+it to join.
+If you don't find the invitation, accept it `here <https://github.com/coala>`__.
 
 Now that you are part of our organization, you can start working on issues.
 If you are familiar with git, you can skip the next section and pick an issue.
 
 Optional. Get Help With Git
 ---------------------------
 
-We use GitHub to manage our repository. If you're not familiar with git, we
-strongly recommend following a tutorial, such as `this one <https://try.github.io/levels/1/challenges/1>`_.
+We use GitHub to manage our repository.
+If you're not familiar with git, we strongly recommend following a tutorial,
+such as `this one <https://try.github.io/levels/1/challenges/1>`_.
 
 We also have a page dedicated to git commands that will help you learn the
 basics: `here <http://coala.io/git>`_.
 
-If there's anything unclear, or you are encountering problems, feel free
-to contact us on `gitter <https://coala.io/chat>`_,
-and we will help you!
-
 Step 3. Picking Up an Issue
 ---------------------------
 
 Now it is time to pick an issue.
-Here is the link that will lead you to `Newcomers issues <https://coala.io/new>`_.
-
-.. note::
-
-    You need to be logged in before you follow the Newcomers issues link.
-
-.. seealso::
-
-    For more information about what bears are, please check the following link: `Writing Native bears <http://api.coala.io/en/latest/Developers/Writing_Native_Bears.html>`_
+Here is the link that will lead you to
+`newcomer issues <https://coala.io/new>`_.
+(You need to be logged in before you follow the Newcomers issues link.)
 
 The easy issues that will help you get started are labeled as
 ``difficulty/newcomer`` and are only there to give you a glimpse of how it is
-to work with us and regarding the workflow.
+to work with us and learning the workflow.
+Look through the list for an issue that you feel comfortable to work on and
+understand what has to be changed without having to ask for help in the chat.
 
-Now pick an issue which isn't assigned, and if you want to fix
-it, then leave a comment that you would like to get assigned. This way
-we don't have multiple people working on the same issue at the same time.
-Now you can start working on it.
+Now pick an issue **which isn't assigned**.
```

Comment:
this doesn't need to be bold, if they try anyway cobot will tell them and they wont loose a lot of time